typescript-light-bulb/main.ts

```diff
@@ -0,0 +1,11 @@
+const $lightBulb = document.querySelector('#light-bulb');
+const $body = document.querySelector('body');
+
+if (!$lightBulb || !$body) throw new Error('No light bulb or body bro');
+const classes = $lightBulb?.classList;
+const classesBody = $body.classList;
```

Comment:
It would be better to separate these into two different `if` statements. While it does work to throw errors, the more specific your error handling, the more clear it is what you need to fix when debugging.